lib/ask/runtime/nuntium_channel.ex

```diff
@@ -74,6 +83,25 @@ defmodule Ask.Runtime.NuntiumChannel do
           :ok
 
         respondent ->
+          channel = respondent_channel(respondent)
+
+          case channel do
+            nil ->
+              # A contact ended, but the channel is missing.
+                # The corresponding ChannelBroker process can't be reached.
+                # This situation occurs when the respondent.session is lost.
+                # Tackled by the channel broker garbage collector.
+              nil
+            _ ->
+              # Should check the state value?
```

Comment:
@ggiraldez, here we're reacting over a Nuntium callback to Surveda. We should check if this state can be different from failed or delivered. I guess the answer is in Nuntium. Any clue? Thanks!!